src/main/java/com/example/processmanagementtool/utlitiy/TemplateMapToJsonConverter.java

```diff
@@ -0,0 +1,29 @@
+package com.example.processmanagementtool.utlitiy;
+
+import com.example.processmanagementtool.domain.template.TemplateProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.r2dbc.postgresql.codec.Json;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.WritingConverter;
+
+import java.util.Map;
+
+@Slf4j
+@WritingConverter
+@RequiredArgsConstructor
+public class TemplateMapToJsonConverter implements Converter<Map<String, TemplateProperty>, Json> {
+    private final ObjectMapper objectMapper;
+
+    @Override
+    public Json convert(Map<String, TemplateProperty> source) {
+        try {
+            return Json.of(objectMapper.writeValueAsString(source));
+        } catch (JsonProcessingException e) {
+            log.error("Error occurred while serializing map to JSON: {}", source, e);
+        }
+        return Json.of("");
```

Comment:
po co to? po 1 nie mozna po 2 wyobraz sobie ze jakims cudem to dziala jak to zdebugujesz? skad bedziesz wiedzial ze "" jest poprawne bo ktos wpisal puste albo zjebalo sie cos w konwerterach, ew jak chcesz mozesz sprawdzic czy jest empty i wywala